### ISSUE TITLE
Meta issue for upcoming version 1.2

### ISSUE THREAD

**Issue by matze** [matze/mtheme#252]
Here's my short list on things that *must* be ready before we can release 1.2

- [x] Check it still works with new and old Beamer versions
- [ ] Fix all issues marked for the 1.2 [milestone](https://github.com/matze/mtheme/milestone/3)

and things that would be nice to have in the next version but could also be postponed

- [ ] Option for thicker font and progress bar as mentioned in #244 

Please comment if you think more fixes or features have to be included.

**Comment by matze** [comment]
Change of plans, we will push version 1.2 immediately and postpone all other issues.
